target/linux/qualcommax/ipq807x/base-files/etc/hotplug.d/ieee80211/11_fix_wifi_mac

```diff
@@ -0,0 +1,18 @@
+[ "$ACTION" == "add" ] || exit 0
+
+PHYNBR=${DEVPATH##*/phy}
+
+[ -n $PHYNBR ] || exit 0
+
+. /lib/functions.sh
+. /lib/functions/system.sh
+
+board=$(board_name)
+
+case "$board" in
+	arcadyan,aw1000)
+		addr=$(cat /sys/class/net/wan/address)
```

Comment:
Are the MAC-s populated in ART?

Reply:
No, there is no mac address from the wireless zone on ART.

Reply:
Hm, I wonder if we should add support for patching the MAC-s in like ath10k has during calibration loading instead of later fixing them up